backend/users/serializers.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth.models import User
+from django.contrib.auth.password_validation import validate_password
+from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
+
+
+class RegisterUserSerializer(serializers.ModelSerializer):
+    email = serializers.EmailField(
+        required=True, validators=[UniqueValidator(queryset=User.objects.all())]
+    )
+
+    password = serializers.CharField(
+        write_only=True, required=True, validators=[validate_password]
+    )
```

Comment:
There is no need to specify validators here, because the email must be unique (this is defined in the new user model). The password is also validated by default (AUTH_PASSWORD_VALIDATORS in the settings file).

Reply:
A field is required by default.